### PR TITLE
Update mint-your-cw20-token.md

### PR DESCRIPTION
**Pull request by Atari-buzzk1LL**
Made some readability adjustments to provide clarity & fixed a typo in JSON for instantiating your own token.

### CODE REVIEW

File: content/en/docs/how-to/mint-your-cw20-token.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ There are some advantages below:
 
 The standard CW20 token is already stored in Fetch.ai network.\
 The code is wrapped based on cw20 [v0.13.2](https://docs.rs/cw20/0.13.2/cw20/index.html) \
-Please check [here]({{< relref "/docs/contract-resources/contract-addresses" >}}) for the more addresses.
+Please check [here]({{< relref "/docs/contract-resources/contract-addresses" >}}) for the token factory addresses.
```
Comment (jhlee-young) [inline content/en/docs/how-to/mint-your-cw20-token.md]: This line guides cw20’s code id, not a factory address. plz leave is as is or update to the token code id.

**Review by Atari-buzzk1LL — COMMENTED** [review]
Updated to fit suggested adjustments by you @jhlee-young

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**